sdk/nodejs/api/utils.ts

```diff
@@ -35,7 +35,7 @@ export function queryBuilder(q: QueryTree[]) {
   })
   query += "}"
 
-  return query.replace(/\s+/g, "")
+  return query.replace(/\s+/g, " ")
```

Comment:
I think we still don't want this since it'll mess up the semantics of the query, not just the syntax. For example if someone tries to do this:

```
echo "foo    bar"
```

it'll get replaced with this:

```
echo "foo bar"
```

Reply:
Just pushed a fix that return a raw query. 
Queries are only sanitize for tests